src/components/modal/components/Profile/utils.ts

```diff
@@ -30,12 +30,26 @@ export function buildProfileFromMetadata(
     (img) => img.tag === "cover"
   );
   const isLens = useLens && lensProfile;
-  const lensCoverPicture = isLens
-    ? { url: getLensCoverPictureUrl(lensProfile), type: "" } // type is not necessary
-    : null;
-  const lensProfileImage = isLens
-    ? { url: getLensProfilePictureUrl(lensProfile), type: "" }
-    : null;
+  let lensCoverPicture = null,
+    lensProfileImage = null;
+  if (isLens) {
+    const lensCoverPictureUrl = getLensCoverPictureUrl(lensProfile);
+    const lensCoverPictureType = lensCoverPictureUrl
+      ? "image/" + lensCoverPictureUrl.split(".").pop()
+      : "";
+    lensCoverPicture = {
+      url: lensCoverPictureUrl,
+      type: lensCoverPictureType
```

Comment:
`type` property is required when validating the form https://github.com/bosonprotocol/interface/blob/3d02a0498c87779fecaabe58e4b3e36939f3b4e0/src/components/product/utils/validationUtils.ts#L18